README.md

```diff
@@ -53,6 +53,7 @@ For tricky use cases, please refer to the [Tips and Tricks](/Docs/TipsAndTricks.
     - Automation analysis: YAFC tries to find objects that can be fully automated. For example, wood in a vanilla game cannot be fully automated because it requires to cut trees.
     - Cost analysis: YAFC assigns a cost to each object. The cost is a sum of logistic actions you need to perform to get that object, using the most optimal recipes. YAFC cost is very useful to quickly compare items and recipes. This cost also helps to find which recipes are suboptimal.
     - Flow analysis: YAFC calculates a base that produces enough science packs for all non-infinite research. It knows how much of everything you will probably need.
+- Load projects from command line (e.g. `$ yafc myproject.yafc "C:\Factorio\data" "C:\Factorio\mods" expensive`, with `expensive` being optional)
```

Comment:
This very specific command line format definitely needs documentation, but I'm not sure whether the readme is the best fit for it. This is highly specific documentation while the rest is a high-level description of YAFC features, so it feels a bit out of place.

Reply:
Usually a command has a 'help' flag to make it show the required/available options.

But  I agree the general README wouldn't be the best place for this kind of documentation. Mentioning that it is possible and linking to `/Docs/CLI.md` would be a better solution.

Reply:
I added the supposed `CLI.md` and linked it from the README.